skyportal/handlers/api/source.py

```diff
@@ -534,6 +540,32 @@ def get(self, obj_id=None):
             description: |
               Arrow-parseable date-time string (e.g. 2020-01-01 or 2020-01-01T00:00:00 or 2020-01-01T00:00:00+00:00).
               If provided, filter by created_at or modified > createdOrModifiedAfter
+          - in: query
+            name: localizationDateobs
+            schema:
+              type: string
+            description: |
+              Event time in isot format
+          - in: query
+            name: localizationName
+            schema:
+              type: string
+            description: |
+              Name of localization / skymap to use
+          - in: query
+            name: localizationCumprob
+            schema:
+              type: number
+            description: |
+              Cumulative probability up to which to include galaxies
+          - in: query
+            name: includeGeojson
+            nullable: true
+            schema:
+              type: boolean
+            description: |
+              Boolean indicating whether to include associated geojson. Defaults to
+              false.
```

Comment:
Why are we extending the source query to include this, instead of making it its own endpoint?  We try to keep the sources relatively lightweight (we've already somewhat failed at that), but it would be good not to make them even heavier.

Reply:
`/api/sources/14xyz/<new_obj_type>` is what we've used for spectra

Reply:
but we need all of this other handling for the parameters right?

Reply:
The parameter handling will be needed, yes.

Reply:
But should it be part of the sources endpoint is what I'm not clear about.

Reply:
I think the only reason to include this in the `source` handler is if you are planning to find "all sources inside a boundary". I'm not sure if there's a more straightforward way to do that in a separate handler...